### PR TITLE
Grammar fix / improvements

### PR DESCRIPTION
**Pull request by Blank3495**
fixed some grammar errors and stuff

### CODE REVIEW

**Comment by tlbl4nk** [comment]
Correction:
line 110: "The total time since the account was logged in"

**Comment by Lautenschlager-id** [comment]
Again, all doc changes requires code changes as well.

File: docs/Client.md [hunk 1]
```diff
@@ -170,7 +170,7 @@
 >| C | `int` | ✔ | The C packet. |
 >| CC | `int` | ✔ | The CC packet. |
 >| f | `function` | ✔ | The function to be triggered when the @C-@CC packets are received. |
->| append | `boolean` | ✕ | True if the function should be appended to the (C, CC) listener, false if the function should overwrite the (C, CC) listener. <sub>(default = false)</sub> |
+>| append | `boolean` | ✕ | `True` if the function should be appended to the (C, CC) listener, `False` if the function should overwrite the (C, CC) listener. <sub>(default = false)</sub> |
```
Comment (Lautenschlager-id) [inline docs/Client.md]:
```suggestion
>| append | `boolean` | ✕ | 'true' if the function should be appended to the (C, CC) listener, 'false' if the function should overwrite the (C, CC) listener. <sub>(default = false)</sub> |
```

File: docs/Client.md [hunk 2]
```diff
@@ -180,7 +180,7 @@
 >| :-: | :-: | :-: | - |
 >| tribulleId | `int` | ✔ | The tribulle id. |
 >| f | `function` | ✔ | The function to be triggered when this tribulle packet is received. |
->| append | `boolean` | ✕ | True if the function should be appended to the (C, CC, tribulle) listener, false if the function should overwrite the (C, CC) listener. <sub>(default = false)</sub> |
+>| append | `boolean` | ✕ | `True` if the function should be appended to the (C, CC, tribulle) listener, `False` if the function should overwrite the (C, CC) listener. <sub>(default = false)</sub> |
```
Comment (Lautenschlager-id) [inline docs/Client.md]:
```suggestion
>| append | `boolean` | ✕ | 'true' if the function should be appended to the (C, CC, tribulle) listener, 'false' if the function should overwrite the (C, CC) listener. <sub>(default = false)</sub> |
```

File: docs/Client.md [hunk 3]
```diff
@@ -286,8 +286,8 @@
 >| :-: | :-: | :-: | - |
 >| playerName | `string` | ✔ | The name of player to be recruited. |
 >
->Sends a recruitment invite to the player.<br>
->![/!\\](http://images.atelier801.com/168395f0cbc.png) Note that this method will not cover errors if the account is not in a tribe or do not have permissions.
+>Sends a tribe invite to a player.<br>
+>![/!\\](http://images.atelier801.com/168395f0cbc.png) Note that this method will not cover errors if the account is not in a tribe or does not have permissions to invite.
```
Comment (Lautenschlager-id) [inline docs/Client.md]:
```suggestion
>![/!\\](http://images.atelier801.com/168395f0cbc.png) Note that this method will not cover errors if the account is not in a tribe or does not have permissions.
```

File: docs/Client.md [hunk 4]
```diff
@@ -338,15 +338,15 @@
 >| message | `string` | ✔ | The message. |
 >
 >Sends a message to a #chat.<br>
->![/!\\](http://images.atelier801.com/168395f0cbc.png) Note that the limit of characters for the message is 255, but if the account is new the limit is set to 80. You must limit it yourself or the bot may get disconnected.
+>![/!\\](http://images.atelier801.com/168395f0cbc.png) Note that the character limit for a message is 255 characters, but if the account is new the limit is 80 characters instead. You must set the limit yourself or the bot may get disconnected.
```
Comment (Lautenschlager-id) [inline docs/Client.md]: What about
```suggestion
>![/!\\](http://images.atelier801.com/168395f0cbc.png) Note that a message has a limit of 80 characters in the first 24 hours after the account creation, and 255 characters later. You must handle the limit yourself or the bot may get disconnected.
```

File: docs/Client.md [hunk 5]
```diff
@@ -355,7 +355,8 @@
 >| message | `string` | ✔ | The message. |
 >
 >Sends a message in the room chat.<br>
->![/!\\](http://images.atelier801.com/168395f0cbc.png) Note that the limit of characters for the message is 255, but if the account is new the limit is set to 80. You must limit it yourself or the bot may get disconnected.
+>![/!\\](http://images.atelier801.com/168395f0cbc.png) Note that the character limit for a message is 255 characters, but if the account is new the limit is 80 characters instead. You must set the limit yourself or the bot may get disconnected.
```
Comment (Lautenschlager-id) [inline docs/Client.md]: What about
```suggestion
>![/!\\](http://images.atelier801.com/168395f0cbc.png) Note that a message has a limit of 80 characters in the first 24 hours after the account creation, and 255 characters later. You must handle the limit yourself or the bot may get disconnected.
```

File: docs/Client.md [hunk 6]
```diff
@@ -364,36 +365,37 @@
 >| message | `string` | ✔ | The message. |
 >
 >Sends a message to the tribe chat.<br>
->![/!\\](http://images.atelier801.com/168395f0cbc.png) Note that the limit of characters for the message is 255, but if the account is new the limit is set to 80. You must limit it yourself or the bot may get disconnected.
+>![/!\\](http://images.atelier801.com/168395f0cbc.png) Note that the character limit for a message is 255 characters, but if the account is new the limit is 80 characters instead. You must set the limit yourself or the bot may get disconnected.
```
Comment (Lautenschlager-id) [inline docs/Client.md]: What about
```suggestion
>![/!\\](http://images.atelier801.com/168395f0cbc.png) Note that a message has a limit of 80 characters in the first 24 hours after the account creation, and 255 characters later. You must handle the limit yourself or the bot may get disconnected.
```

File: docs/Client.md [hunk 7]
```diff
@@ -364,36 +365,37 @@
...
 >### client:sendWhisper ( targetUser, message )
 >| Parameter | Type | Required | Description |
 >| :-: | :-: | :-: | - |
 >| message | `string` | ✔ | The message. |
->| targetUser | `string` | ✔ | The user to receive the whisper. |
+>| targetUser | `string` | ✔ | The user who will recieve the whisper. |
 >
 >Sends a whisper to an user.<br>
->![/!\\](http://images.atelier801.com/168395f0cbc.png) Note that the limit of characters for the message is 255, but if the account is new the limit is set to 80. You must limit it yourself or the bot may get disconnected.
+>![/!\\](http://images.atelier801.com/168395f0cbc.png) Note that the character limit for a message is 255 characters, but if the account is new the limit is 80 characters instead. You must set the limit yourself or the bot may get disconnected.
```
Comment (Lautenschlager-id) [inline docs/Client.md]: What about
```suggestion
>![/!\\](http://images.atelier801.com/168395f0cbc.png) Note that a message has a limit of 80 characters in the first 24 hours after the account creation, and 255 characters later. You must handle the limit yourself or the bot may get disconnected.
```

File: docs/Client.md [hunk 8]
```diff
@@ -364,36 +365,37 @@
...
+>Sets the community the bot will connect to.<br>
 >![/!\\](http://images.atelier801.com/168395f0cbc.png) This method must be called before the [start](Client.md#clientstart--self-tfmid-token-).
 >
 ---
 >### client:setTribeMemberRole ( memberName, roleId )
 >| Parameter | Type | Required | Description |
 >| :-: | :-: | :-: | - |
 >| memberName | `string` | ✔ | The name of the member to get the role. |
->| roleId | `int` | ✔ | The role id. (starts in 0, for the initial role. Increases until the Chief role) |
+>| roleId | `int` | ✔ | The role id. (starts from 0, the initial role, increases until the Chief role) |
```
Comment (Lautenschlager-id) [inline docs/Client.md]:
```suggestion
>| roleId | `int` | ✔ | The role id. (starts from 0, the initial role, and goes until the Chief role) |
```

File: docs/Client.md [hunk 9]
```diff
@@ -364,36 +365,37 @@
...
 >
 >Sets the role of a member in the tribe.<br>
->![/!\\](http://images.atelier801.com/168395f0cbc.png) Note that this method will not cover errors if the account is not in a tribe or do not have permissions.
+>![/!\\](http://images.atelier801.com/168395f0cbc.png) Note that this method will not cover errors if the account is not in a tribe or do not have permissions to change member ranks.
```
Comment (Lautenschlager-id) [inline docs/Client.md]:
```suggestion
>![/!\\](http://images.atelier801.com/168395f0cbc.png) Note that this method will not cover errors if the account is not in a tribe or do not have permissions.
```

File: docs/Client.md [hunk 10]
```diff
@@ -101,13 +101,13 @@
 >
 ---
 >### client:connectionTime (  )
->Gets the total time of the connection.
+>Gets the total time since the account was connected.
```
[the remark this answers is not in the document]
Comment (Blank3495) [reply docs/Client.md]: o ok ill make correction

File: docs/Client.md [hunk 11]
```diff
@@ -159,7 +159,7 @@
 >| C | `int` | ✔ | The C packet. |
 >| CC | `int` | ✔ | The CC packet. |
 >| f | `function` | ✔ | The function to be triggered when the @C-@CC packets are received. |
->| append | `boolean` | ✕ | True if the function should be appended to the (C, CC) listener, false if the function should overwrite the (C, CC) listener. <sub>(default = false)</sub> |
+>| append | `boolean` | ✕ | `True` if the function should be appended to the (C, CC) listener, `False` if the function should overwrite the (C, CC) listener. <sub>(default = false)</sub> |
```
[the remark this answers is not in the document]
Comment (Blank3495) [reply docs/Client.md]: >:( fine i was totally not doing python here

File: docs/Client.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (Blank3495) [reply docs/Client.md]: We've decided to keep the change

[the remark this answers is not in the document]
Comment (Lautenschlager-id) [reply docs/Client.md]: We've decided to keep the change

**Pull request merged**